sensorlib/src/sensor/SemanticLidarSensor.py

```diff
@@ -286,6 +288,9 @@ def compute_closest_object_id(self, hitpoint, scene_objects, geometry_associatio
 
         The threshold prevents association between a point and object which are very far apart.
         """
+        #check if there are no objects around
+        if not scene_objects:
+            return None
```

Comment:
let's add this to the very beginning of the calls. Return empty if there was no objects detected in the beginning
https://github.com/usdot-fhwa-stol/carma-utils/blob/4628b18c4b63858ae919fce0119183dd9b5002c6/sensorlib/src/sensor/SemanticLidarSensor.py#L85
https://github.com/usdot-fhwa-stol/carma-utils/blob/4628b18c4b63858ae919fce0119183dd9b5002c6/sensorlib/src/sensor/SemanticLidarSensor.py#L88